output_template/python/usb_camera_demo.py

```diff
@@ -317,6 +369,9 @@ def run(model, config_file):
     if config.TASK == "IMAGE.SEMANTIC_SEGMENTATION":
         run_sementic_segmentation(config)
 
+    if config.TASK == "IMAGE.KEYPOINT_DETECTION":
```

Comment:
please make this a func table

```
TASK_HANDLERS = {
    "IMAGE.KEYPOINT_DETECTION": run_keypoint_detection,
   ...
}

TASK_HANDLERS[config.TASK](config)
```

Reply:
Good idea! I have changed it and tested it manually. 😄